### DISCUSSION TITLE
How to Export file .png language  JavaScript file size 9x18cm dpi 300

### DISCUSSION THREAD

**Discussion by gitdevdeveloper** [community/community#63689] [Codespaces]
I want to save my image from canvas with image size 9x18 cm. Downloaded it got dpi 300 i tried it all and all the knowledge i have pls tell me the guide

**Comment by proksh** [comment]
To save an image from a canvas with specific dimensions (9x18 cm) at a DPI (Dots Per Inch) of 300, you need to consider the pixel dimensions of the image. Here's a step-by-step guide:

- **Calculate the pixel dimensions:**
At 300 DPI, 1 inch equals 300 pixels. To convert centimeters to inches, you can use the conversion factor: 1 inch = 2.54 cm.

Width in pixels:
9cm×(300pixels/2.54cm) = 1062.99 or approximately 1063 pixels

Height in pixels:
18cm×(300pixels/2.54cm)  = 2125.98 or approximately 2126 pixels

So, you need to set your canvas size to 1063x2126 pixels for a 9x18 cm image at 300 DPI.

- **Set the canvas size:**
Depending on the tool or software you're using, there might be an option to set the canvas size directly. Ensure your canvas dimensions are set to 1063x2126 pixels.

- **Draw or place your content on the canvas.**

- **Save the image:**
When saving the image, ensure that the resolution is set to 300 DPI. The method to do this will vary depending on the software you're using. For instance, in Adobe Photoshop, when saving the image, there's an option to set the DPI.

- **Check the image properties:**
After saving, you can check the image's properties to confirm the pixel dimensions and resolution (DPI). Ensure it matches the values you intend.

- **Download:**
If you're working on an online platform or tool, download the image to your local storage.

- **Print Test:**
Before getting a bulk print or using the image for critical purposes, always do a print test to ensure it meets your requirements in terms of clarity, color accuracy, and size.

Remember that DPI is crucial when you plan to print the image. For on-screen display, pixel dimensions are more relevant. A 300 DPI resolution is excellent for high-quality prints.
